lib/main.coffee

```diff
@@ -49,8 +38,8 @@ module.exports =
         filePath = textEditor.getPath()
         command = @executablePath
         ruleset = @rulesets
-        if @projectRules
-          rulesetPath = helpers.find(filePath, 'ruleset.xml')
+        if /^[a-z0-9]+\.xml$/gi.test(@rulesets)
```

Comment:
This isn't going to work like you are thinking, the way you have it currently written if there is _any_ `foo.xml` anywhere in the ruleset it will attempt to find a file with a name of the entire `rulesets` value. What happens if somebody does something like `cleancode,myrules.xml`?

Also the way this is written if it does find a file somehow, it will override the entire setting with whatever it found for that first file.